Milestone4/OfflineStudentManagementSystem/Model/interfaces/studentDetailsInterface.ts

```diff
@@ -0,0 +1,7 @@
+export interface IStudentDetails {
+  mark1: number;
+  mark2: number;
+  mark3: number;
+  total: number;
+  grade: string;
```

Comment:
Can we use subject names instead of mark1, mark2 etc.,

Reply:
resolved